### DISCUSSION TITLE
Welcome to vertex-ai-mlops Discussions!

### DISCUSSION THREAD

**Discussion by statmike** [statmike/vertex-ai-mlops#7] [Announcements]
## 👋 Welcome!
Let's use Discussions as a place to connect!
  * Ask questions
  * Share ideas
  * Engage with other community members
  * Build a community

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rafal-wasowski** [comment]
Hi Mike,

Great idea, I hope the community will grow in time.

I've been using BQML quite extensively since its release but Vertex AI is still something I'm just dabbling with. Your YT tutorials were a great help with getting started 👍

-----
Rafal

**Reply by statmike** [reply]
Thank you!  Let me know how it goes and if you have ideas for new content that might help.  BQML is an amazing way to accelerate ML!
